adam/curriculum_to_html.py

```diff
@@ -0,0 +1,184 @@
+import os
+from typing import List
+
+from attr import attrs
+
+from adam.experiment import InstanceGroup
+from adam.language.dependency import LinearizedDependencyTree
+from adam.perception import PerceptualRepresentation
+from adam.perception.developmental_primitive_perception import (
+    DevelopmentalPrimitivePerceptionFrame,
+)
+from adam.situation.high_level_semantics_situation import HighLevelSemanticsSituation
+
+
+@attrs(frozen=True, slots=True)
+class CurriculumToHtml:
+    """
+    Class to turn an `InstanceGroup` into an html document
+    """
+
+    @staticmethod
```

Comment:
Why static?  Maybe rename the class to `CurriculumToHtmlDumper` and have the user instantiate one when they want to use it?

(the advantage of this is that we may want to add configuration options to the dumper in the future)